src/main/play-doc/developer/CreatingBundles.md

```diff
@@ -1,23 +1,44 @@
 
-Bundles fall broadly into one of two categories
-
-* those that don't use Docker; and
-* those that do.
-
-Accordingly we use the name "universal" to refer to non-Docker based bundles, and "docker" for when using a Dockerfile. In addition, Docker images can be run from "universal" bundles. The following sections cover each of these scenarios:
+ConductR supports several different types of bundle components.
+  * OCI bundle components (introduced in 2.1) package an entire container in the bundle. They can be produced from Docker images. They run inside a container using `runc`.
+  * Universal bundle components run directly on the host.
+  * Docker bundle components build a provided `Dockerfile` and run the resulting image
+  
+In addition, Docker images can be run from "universal" bundles. The following sections cover each of these scenarios:
 
+* [Producing an OCI bundle with Docker](#producing-an-oci-bundle-with-docker)
 * [Producing a universal bundle without Docker](#producing-a-universal-bundle-without-docker)
 * [Producing a universal bundle that runs a Docker image](#producing-a-universal-bundle-that-runs-a-docker-image)
 * [Producing a docker bundle that uses a Dockerfile](#producing-a-docker-bundle-that-uses-a-dockerfile)
 
+## Producing an OCI bundle with Docker
+ConductR is capable of running images that have been exported from Docker. To do this, use the `bndl` tool to convert a docker image into a ConductR bundle. Behind the scenes, we're using the standard OCI Image format so you can be sure your images are free of vendor lock-in. Additionally, this approach doesn't require Docker to be installed in production; ConductR provides everything necessary to run these images.
+
+The following command will load your docker image directly into ConductR.
+
+```bash
+docker save my-image | bndl | conduct load
+```
+
+If you'd rather save the resulting bundle to a file, you can do this using `bndl -o <filename>`.
+```bash
+docker save my-image | bndl -o my-bundle.zip
+```
+
+
+We attempt to extract as much information from the your docker image as possible. Any `EXPOSE` directives in your `Dockerfile` will automatically create corresponding service endpoints in `bundle.conf`. Support for persistent mounting of `VOLUME` directives is planned.
+
+Docker images can vary in size significantly and this can affect development/deployment time and consume additional system resources. Because of this, we recommend that you build your images ontop of a light-weight base image. [Alpine](https://hub.docker.com/_/alpine/) is a great choice as it's only 5MB in size. 
+
```

Comment:
I think we should go further here and recommend `openjdk/8-jre-alpine` as the base image to use.

(81.4MB in size)